app/models/recipe.rb

```diff
@@ -0,0 +1,8 @@
+class Recipe < ApplicationRecord
+  belongs_to :user
+  has_many :recipe_foods
+  has_many :foods
```

Comment:
- [ ] According to your migration and the ERD diagram. a Recipe should not have a direct relationship to the "foods" table. you can use the recipe_foods to connect the recipe and foods.

```suggestion
  has_many :foods, through: :recipe_foods
```